.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,107 @@
+---
+name: 🐛 Bug Report
+description: Create a report to help us improve
+labels: ["bug", "needs-triage"]
+
+body:
+- type: markdown
+  attributes:
+    value: >
+      **Thank you for wanting to report a bug in pyhf!**
+
+
+      ⚠
+      Verify first that your issue is not [already reported on
+      GitHub][issue search].
+
+
+      [issue search]: https://github.com/scikit-hep/pyhf/issues?q=is%3Aopen+is%3Aissue+label%3Abug
+
+- type: textarea
+  attributes:
+    label: Summary
+    description: Explain the problem briefly below.
+    placeholder: >-
+      When I try to do X with pyhf and the following workspace, Y breaks or
+      Z happens in an unexpected manner.
+      Here are all the details I know about this problem.
+  validations:
+    required: true
+
+- type: textarea
+  attributes:
+    label: pyhf Version
+    description: >-
+      Paste verbatim output from `pyhf --version` below, under
+      the prompt line. Please don't wrap it with tripple backticks &mdash; your
+      whole input will be turned into a code snippet automatically.
```

Comment:
I am assuming that users with editable installs who checkout different versions of `pyhf` will generally understand the system enough so that this is not an issue, but in principle the output of `pyhf --version` may not match the actual version in use (this is for example the case for me right now).

Reply:
@alexander-held You just saying that if you are locally installing `pyhf` and not using a supported release from PyPI then you might get something like

```console
$ pyhf --version
pyhf, version 0.6.3.dev17
```

? Or do you mean something in addition to this?

Reply:
I mean something closely related, yes. I won't get the dev tags if I clone the repo, check out tag `v0.6.2`, `pip install -e .`, and then check out the master branch. `pyhf --version` will show `pyhf, version 0.6.2` nevertheless, even though I am using a newer version. This is so minor though that it is probably not worth any changes.

Reply:
Ah gotcha yeah. :+1: I think this is probably okay, as you're like our ultimate power user and you know what you're doing here. I'm more assuming that people who are reporting bugs are installing directly from PyPI.

Though I'll mention here that when you report bugs in the future feel free to just give whatever information is the most relevant here (e.g. git commit hash) or something.